test/test_pty.rs

```diff
@@ -90,3 +93,81 @@ fn test_open_ptty_pair() {
     let slave_fd = open(Path::new(&slave_name), O_RDWR, stat::Mode::empty()).unwrap();
     assert!(slave_fd > 0);
 }
+
+#[test]
+fn test_openpty() {
+    let pty = openpty(None, None).unwrap();
+    assert!(pty.master > 0);
+    assert!(pty.slave > 0);
+
+    // writing to one should be readable on the other one
```

Comment:
Can you capitalize the first work in your line comments? Please do that to all of them as well.